### PR TITLE
Fix examples nits

### PR DESCRIPTION
**Pull request by beaufortfrancois**
FIX: #99

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -506,50 +509,51 @@
...
-          .then((blob) => {
-            console.log("Photo taken: " + blob.type + ", " + blob.size + "B")
-
-            const imageTag = document.querySelector('input[type="img"]');
-            imageTag.src = URL.createObjectURL(blob);
-          })
-          .catch((err) => {
-            console.error("takePhoto() failed: ", e);
-          });
+        imageCapture.takePhoto().then(blob => {
```
Comment (miguelao) [inline index.html]: I'm trying to put the .then() in a new line,
indented by four spaces, like in l.488, 599 etc. 
Let's pick one and be consistent?

Comment (beaufortfrancois) [inline index.html]: You're right. Sorry for that.

**Comment by beaufortfrancois** [comment]
@miguelao I've updated the PR. WDYT?

**Comment by miguelao** [comment]
I sayz good! Merging it.

**Pull request merged**